accelerator/cuda_accelerator.py

```diff
@@ -234,8 +234,9 @@ def _lazy_init_class_dict(self):
             op_builder_dir = self.op_builder_dir()
             op_builder_module = importlib.import_module(op_builder_dir)
             for _, module_name, _ in pkgutil.iter_modules([os.path.dirname(op_builder_module.__file__)]):
-                # avoid self references
-                if module_name != 'all_ops' and module_name != 'builder' and module_name != 'cpu':
+                # avoid self references and skip other backends
+                if module_name != 'all_ops' and module_name != 'builder' and module_name != 'no_impl' \
+                    and module_name != 'cpu' and module_name != 'npu':
```

Comment:
```suggestion
                if module_name != 'all_ops' and module_name != 'builder' and module_name not in [no_impl', 'cpu', 'npu']:
```

Reply:
How about make all module_names we want to exclude into a list?

Reply:
I remember `cpu` is included because `cpu/` is a subdirectory of op_builder and get reflected.   I think rather than make a list to include `cpu` and `npu`, there are two options:
1. Exclude module names that is _directory_, so subdirectory created for other accelerators would be excluded.
2. Move CUDA op_builder under `cuda/` subdirectory and reflect op builder from that subdirectory, this would also make directory structure more clean.

Reply:
I used to think about Option 1. But exclude all directory may exclude directories that contains ops. For example, cuda may add a group of similar ops and put them in a directory. 

And Move CUDA op_builder under cuda/ maybe a better choice, I didn't do this to avoid too may code change. 
I will use Option2 for next commit.

Reply:
After thinking about these two options, I want to choose Option1.